spring-petclinic/pom.xml

```diff
@@ -195,13 +189,19 @@
                  read this as the JBoss stack of the Java EE APIs). You can actually
                  use this stack with any version of JBoss EAP that implements Java EE. -->
             <dependency>
+                <groupId>org.jboss.spec</groupId>
```

Comment:
I think we can remove this BOM since jboss-javaee-7.0-eap-with-tools includes it